### DISCUSSION TITLE
Install error

### DISCUSSION THREAD

**Discussion by milofank** [wevm/wagmi#1249] [Q&A]
When I install wagmi, this is the error I read in the CLI
D:\TestSolidity\test>npm i wagmi
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Fank's\AppData\Local\npm-cache\_logs\2022-11-07T14_54_50_959Z-debug-0.log

**Comment by milofank** [comment]
![Sans titres](https://user-images.githubusercontent.com/117664856/200363481-d28cdd3b-b0bc-46ba-bac5-396e78880ce3.png)

**Reply by tmm** [reply]
Have you tried this? https://stackoverflow.com/questions/49620780/cant-install-any-npm-package-error-4058/49620827#49620827

**Reply by milofank** [reply]
The same error happening

**Reply by milofank** [reply]
I solved the problem
'Git' was not installed. When I install the 'git', I succeded install wagmi
